containerd/containerd.go

```diff
@@ -114,7 +123,7 @@ var ContainerdCommand = cli.Command{
 		} else {
 			f = factory.NewFromConfigs(kernel, initrd, nil)
 		}
-		sv, err := supervisor.New(stateDir, containerdDir, f)
+		sv, err := supervisor.New(stateDir, containerdDir, f, defaultCpu, defaultMemory)
```

Comment:
could you directly use `context.GlobalInt("default_cpus")` and
`context.GlobalInt("default_memory")` here?